eos/form-factors/parametric-bgl1997-impl.hh

```diff
@@ -197,27 +211,48 @@ namespace eos
     }
 
     double
-    BGL1997FormFactors<BToDstar>::t_1(const double & /*s*/) const
+    BGL1997FormFactors<BToDstar>::t_1(const double & s) const
     {
-        return 0.0;  //  TODO
+        // resonances for 1^-, which have overlap with the tensor current
+        const double blaschke = _z(s, 6.329 * 6.329) * _z(s, 6.910 * 6.910) * _z(s, 7.020 * 7.020);
```

Comment:
Why are the resonances masses hard-coded here?

Reply:
Copy and paste from the other BGL form factors. Since this is a number of terms, we would need to store the information within a set in the corresponding ``BToDstar`` process trait.

Reply:
Yes exactly. This can be part of the post merge cleaning commit if you prefer